### PR TITLE
Restaurant dishes

### PR DESCRIPTION
**Pull request by NegativeKarma**
# PT-Link: https://www.pivotaltracker.com/story/show/154174339

## Description:
Implement dish category as an associated model of menu and add dishes belonging to the dish category

## What we learned:
* How to nest each iterations method for models that have an association with each other
* Use the span element as in-line element

## Screenshot:
restaurant dishes on restaurant page

### CODE REVIEW

File: features/step_definitions/basic_steps.rb [hunk 1]
```diff
@@ -22,6 +22,10 @@
   expect(current_path).to eq root_path
 end
 
+Then("show me the page") do
+  save_and_open_page
+end
+
```
Comment (SophieSjoberg) [inline features/step_definitions/basic_steps.rb]: Remove this before we can merge. We don't want this in our deployed code. :)

**Comment by SophieSjoberg** [comment]
Me and Mariza had a look, great job! 
Some notes: Have a look at your feature files - do you really need to have two separate ones?
Also please revisite the user story for dish-category, is that really reflective of what the user wants? 
Please change price - to float instead of integer. 👍

File: features/step_definitions/basic_steps.rb (hunk 1 above)
Comment (NegativeKarma) [reply features/step_definitions/basic_steps.rb]: Ok

File: app/models/dish.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class Dish < ApplicationRecord
```
Comment (tochman) [inline app/models/dish.rb]: Why do you choose this name? Will you only sell food or will you also serve drinks and other prospects?

File: db/migrate/20180116133813_add_price_to_dish.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddPriceToDish < ActiveRecord::Migration[5.1]
+  def change
+    add_column :dishes, :price, :float
```
Comment (tochman) [inline db/migrate/20180116133813_add_price_to_dish.rb]: Why 2 migrations? Removing and adding price?

Comment (NegativeKarma) [reply db/migrate/20180116133813_add_price_to_dish.rb]: Because we needed to change the price from integer to float

Comment (NegativeKarma) [reply db/migrate/20180116133813_add_price_to_dish.rb]: We probably could have solved with one migration but we felt safer to first remove price as a integer and then add it back as a float

File: app/models/dish.rb (hunk 2 above)
Comment (NegativeKarma) [reply app/models/dish.rb]: We're changing it to product and product_category

**Pull request merged**